src/components/action-bar/action-bar.tsx

```
@@ -187,6 +237,16 @@
         }
     };
 
+    private get tooltipLabel() {
+        return this.actionBarIsShrunk
```

ESLint

[error] Unexpected newline between test and consequent of ternary expression.

(multiline-ternary)